landscape.yml

```diff
@@ -48,6 +48,13 @@ landscape:
       - subcategory:
         name: Ethernet
         items:
+          - item:
+            name: Cisco
+            homepage_url: https://cisco.com/
+            logo: cisco.svg
+            crunchbase: https://crunchbase.com/organization/cisco
```

Comment:
https://www.crunchbase.com/organization/cisco

Reply:
Ok, it seems as if hte checks are a bit too pedantic, crunchbase works with or without the www.  I can, however, just add the www to the URL.  We should also add cisco to optics.  What is the best way to do this?

Reply:
I have added www to the crunchbase URL and added a second_path for cisco for the optics section.